tests/python/rest_api/test_task_data.py

```diff
@@ -0,0 +1,2247 @@
+import io
+import json
+import math
+import os
+import os.path as osp
+import zipfile
+from collections import Counter
+from collections.abc import Iterable, Sequence
+from copy import deepcopy
+from functools import partial
+from http import HTTPStatus
+from itertools import chain, groupby, product
+from tempfile import TemporaryDirectory
+from typing import Any, Optional
+
+import numpy as np
+import pytest
+from cvat_sdk import exceptions
+from cvat_sdk.api_client import models
+from cvat_sdk.core.helpers import get_paginated_collection
+from deepdiff import DeepDiff
+from PIL import Image
+from pytest_cases import parametrize
+
+import shared.utils.s3 as s3
+from shared.tasks.enums import _SourceDataType
+from shared.tasks.interface import ITaskSpec
+from shared.tasks.utils import parse_frame_step
+from shared.utils.config import get_method, make_api_client, patch_method, post_method
+from shared.utils.helpers import (
+    generate_image_file,
+    generate_image_files,
+    generate_manifest,
+    generate_video_file,
+    read_video_file,
+)
+
+from ._test_base import _TestTasksBase
```

Comment:
It's better to replace relative imports with absolute for consistency. As i see all other imports here are from the root